analytics_data_api/v0/models.py

```diff
@@ -2,11 +2,12 @@
 
 from django.conf import settings
 from django.db import models
-from django.db.models import Sum
+from django.db.models import Count, Sum
 # some fields (e.g. Float, Integer) are dynamic and your IDE may highlight them as unavailable
-from elasticsearch_dsl import Date, DocType, Float, Integer, Q, String
+from elasticsearch_dsl import Date, DocType, Float, Integer, Q, String  # pylint: disable=no-name-in-module
```

Comment:
Yes it was, which confused me, since I wasn't aware of this causing failures before.